docs/advanced-usage/access-control/voms-proxy/index.md

```diff
@@ -0,0 +1,65 @@
+# Voms-proxy
+
+If your workflow needs special permissions that your Virtual Organization entitles you to, the VOMS client can create a proxy certificate for your authentication.
+
+## Testing your credentials
+
+Before moving to REANA, check that your credentials are in order.
+
+```console
+# Login to lxplus
+$ ssh johndoe@lxplus.cern.ch
+# Make sure userkey.pem is read/write only by the owner
+$ ls -l ~/.globus
+-rw-r--r--. usercert.pem
+-r--------. userkey.pem
+
+# Let us create a proxy certificate with the VO cms
+$ voms-proxy-init --voms cms
+Enter GRID pass phrase for this identity:
+
+Contacting voms2.cern.ch:15002 [/DC=ch/DC=cern/OU=computers/CN=voms2.cern.ch] "cms"...
+Remote VOMS server contacted succesfully.
+
+
+Created proxy in /tmp/x509up_u131816.
+
+Your proxy is valid until Wed Apr 01 00:43:08 CEST 2020
+```
+
+## Uploading secrets
+
+To create the proxy certificate the VOMS client needs access to your user certificate `usercert.pem`, encrypted private key `userkey.pem` and GRID pass phrase. For REANA the pass phrase needs to be encoded in Base64. To encode it exchange `mygridpassphrase` for your GRID pass phrase and perform
+
+```console
+$ echo 'mygridpassphrase' | base64
+bXlncmlkcGFzc3BocmFzZQo=
+```
+
+Upload these three secrets to REANA. In addition the client needs information about your virtual organization. This is communicated by the environment variable `VOMSPROXY_VO` and can also be added to the secrets:
```

Comment:
Same here, "the client needs" it might even sound confusing... I would say REANA needs in general, as we do the orchestration of the integration with the VOMS client.